sigs.yaml

```diff
@@ -4028,7 +4028,7 @@ committees:
     - github: natherz97
       name: Nathan Herz
       company: Amazon
```

Comment:
Do you also need to update this?

Reply:
My plan was to hold off until I have my next steps finalized and update it then!